packages/shared/src/components/checklist/NotificationChecklistStep.tsx

```diff
@@ -1,23 +1,19 @@
-import React, { ReactElement, useContext } from 'react';
+import React, { ReactElement } from 'react';
 import { ChecklistStepProps } from '../../lib/checklist';
 import { Button } from '../buttons/Button';
 import { ChecklistStep } from './ChecklistStep';
 import BellIcon from '../icons/Bell';
-import NotificationsContext from '../../contexts/NotificationsContext';
 import { NotificationPromptSource } from '../../lib/analytics';
+import { useEnableNotification } from '../../hooks/useEnableNotification';
 
 const NotificationChecklistStep = (props: ChecklistStepProps): ReactElement => {
-  const { onTogglePermission } = useContext(NotificationsContext);
+  const { onEnable } = useEnableNotification({
```

Comment:
What's the different between `onEnable` and `onTogglePermission`?

Reply:
The `onEnable` will use the `onTogglePermission` but will also include tracking and everything else.